Configurations.md

```diff
@@ -1047,6 +1047,13 @@ fn lorem() -> usize {
 
 See also: [`tab_spaces`](#tab_spaces).
 
+## `hex_literal_case`
+
+Change the case of the letters in hexadecimal literal values
+
+- **Default value**: `Ignore`
+- **Possible values**: `ToUpper`, `ToLower`
+- **Stable**: No
```

Comment:
Couple minor things I'd like to tweak here. Let's adjust the text to reflect the fact that it can _set_ vs. implying it will change.

Instead of `Ignore`, we should use `Preserve` to be consistent with variants in our other configuration options, and then let's drop the `To` prefix on the other variants.

```suggestion
Control the case of the letters in hexadecimal literal values

- **Default value**: `Preserve`
- **Possible values**: `Upper`, `Lower`
- **Stable**: No
```